src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle {
+  public shape = 'triangle';
```

Comment:
I would suggest creating a type for all the colors and shapes
like type Color = 'red' | 'blue' | 'green'
 You can also create an enum instead of type so that you would have autofill